packages/p2p/src/components/my-ads/toggle-ads.jsx

```diff
@@ -22,10 +24,18 @@ class ToggleAds extends Component {
 
     state = {
         is_enabled: this.props.is_enabled,
+        error     : '',
     }
 
-    setEnabled (is_enabled) {
-        this.setState({ is_enabled });
+    handleToggle = () => {
+        const is_active = this.state.is_enabled ? 0 : 1;
+        this.setState({ is_enabled: !this.state.is_enabled });
+
+        requestWS({ p2p_agent_update: 1, is_active }).then((response) => {
+            if (response.error) {
+                this.setState({ error: response.error.message });
```

Comment:
I think after this we should update the is_enabled state again to make sure it's the correct value